04_quicksort/javascript/01_loop_sum.js

```diff
@@ -1,11 +1,34 @@
-'use strict';
+const arr = [1, 2, 3, 4];
 
-function sum(arr) {
-  let total = 0;
-  for (let x = 0; x < arr.length; x++) {
-    total += arr[x];
-  }
-  return total;
-}
+/**
+ * Sums values in array by loop "for"
+ * @param {Array} arr Array of numbers
+ * @return {number} Sum of the numbers
+ */
+const sumLoop = ( arr ) => {
+    const newArr = Array.prototype.slice.call( arr );
+    let result = 0;
 
-console.log(sum([1, 2, 3, 4])) // 10
+    for ( let i = 0; i < newArr.length; i++ ) {
+        result += newArr[i];
+    }
+
+    return result;
+};
+
+/**
```

Comment:
Can you please put this code in a new file `01_loop_sum_reduce_version.js`? I'm trying to keep each file as short as possible, to not intimidate people looking at the code.